src/App.js

```diff
@@ -1,29 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
```

Comment:
- [ ] Hello and nice job on CRA of the app calculator but the `app.js` should be a **class-based component**

Reply:
Thank you @dasileker , I have made the requested changes :+1: